options-gen/testdata/case-03-interfaces/options_generated.go

```diff
@@ -2,6 +2,8 @@
 package testcase
 
 import (
+	"fmt"
```

Comment:
@kazhuravlev та самая странная бага с дублированием импортов, wtf

Reply:
@kazhuravlev, please, try to repeat this bug locally

Reply:
I suppose that this is a bug in `imports`. See a several [examples](https://go.dev/play/p/QEVFgl-HPhb) I wrote. 

For resolve the bug - just remove `"fmt"` from `options.tpl`. codegen will automatically add all necessary imports. But this removement may cause some user cases. To avoid that you should add an alias to `"fmt"` in `options.tpl` like that 
```
import (
 optionsFmt461e464ebed9 "fmt"
)
```

Reply:
https://github.com/golang/go/issues/55982

Reply:
I could return my logic with set of imports, but decided to add alias.
Thanks